Driverless AI/risk-assessment-tools-in-the-world-of-ai-the-social-sciences-and-humanities/risk-assessment-tools-in-the-world-of-ai-the-social-sciences-and-humanities.md

```diff
@@ -0,0 +1,475 @@
+
+  Take the case of Glenn Rodríguez. An inmate at the Eastern Correctional Facility in upstate New York, Mr. Rodríguez was denied parole last year despite having a nearly perfect record of rehabilitation. The reason? A high score from a computer system called Compas. The company that makes Compas [considers the weighting of inputs to be proprietary information](http://washingtonmonthly.com/magazine/junejulyaugust-2017/code-of-silence/).”[9] As a result, Mr. Rodríguez wasn’t able to cross-examine the score (evidence) used against his parole denial. 
+
+The above is a clear example of how AI is deciding whether someone will receive parole while not allowing a defendant to cross-examine the evidence. For legal practitioners and other social scientists, this will be in clear violation of the confrontation clause found in the constitution. The confrontation clause is as follows: 
+
+- “The Confrontation Clause of the [Sixth Amendment to the United States Constitution](https://en.wikipedia.org/wiki/Sixth_Amendment_to_the_United_States_Constitution) provides that "in all criminal prosecutions, the accused shall enjoy the right…to be confronted with the [witnesses [evidence]](https://en.wikipedia.org/wiki/Witness) against him." Generally, the right is to have a face-to-face confrontation with witnesses who are offering testimonial evidence against the accused in the form of cross-examination during a trial. The [Fourteenth Amendment](https://en.wikipedia.org/wiki/Fourteenth_Amendment_to_the_United_States_Constitution) makes the right to confrontation applicable to the states and not just the federal government.[1] The right only applies to criminal prosecutions, not civil cases or other proceedings.”[10]
+
+As of now, it isn't very easy to explain an AI model's predictions. And for the most part, the explanations of these AI models cannot be disclosed because they are protected by law. A lack of explanation of how predictions are derived will prevent the recidivism AI model's use if the predictions can't be explained; cross-examination will not be possible. This lack of explainability hasn't stopped courtrooms from using AI models to determine who is incarcerated and for how long. 
+To address this problem, AI makers and law practitioners should discuss and work together when building recidivism models. The collaboration will allow for an understanding of why such models will be violating the law and how we can perhaps work around the problem of a lack of explainability. This joint work can also lead to the often practice to consult law practitioners before using models that can have substantial legal consequences. In our case, consulting legal practitioners can allow for a deeper understanding of how perhaps our model can affect the confrontations clause or other parts of the law. 
+
+**Problem 2: Fair and unfair models will still be wrong in the legal field**
```

Comment:
Please remove the "I" from the paragraph.

Reply:
@Castro38 I am not able to see the "I" you specify

Reply:
thanks